scripts/visual-active-effects.mjs

```diff
@@ -36,17 +36,32 @@ export class VisualActiveEffects extends Application {
       // Set up the various text (intro and content).
       const desc = entry.effect.flags["dfreds-convenient-effects"]?.description;
       const data = entry.effect.flags[MODULE]?.data ?? {};
+      
+      // Get the effect rollData to populate enrichers within the descriptions.
+      let rollData = {};
+      if (entry.effect.origin) {
+        const origin = await fromUuid(entry.effect.origin);
```

Comment:
Since all origins should hopefully be in the world, it should be sufficient to use the `fromUuidSync` variant. Note that this will throw an error if the uuid points to an embedded document inside a compendium (e.g, an item on an actor in a compendium), so a `try/catch` would be required.

Reply:
When actors with passive effects are pulled directly from a compendium and placed into the canvas their uuids will point to the item on the actor in the compendium. Some modules attempt to correct this behavior however.